lib/bot.js

```diff
@@ -373,6 +404,20 @@ class Bot {
       return match;
     });
 
+    // Webhooks first
+    const webhook = this.findWebhook(channel);
+    if (webhook) {
+      logger.debug('Sending message to Discord via webhook', withMentions, channel, '->', `#${discordChannel.name}`);
+      webhook.client.sendMessage(
+        withMentions,
+        {
+          username: author,
+          text,
+          avatarURL: this.getDiscordAvatar(author)
+        }).catch(logger.error);
+      return;
+    }
+
```

Comment:
… I also failed to notice that this seems to discard the custom formatting when sending with a webhook; I think the 'Webhooks first' code should go after the `const withAuthor…` bit, here, and use `withAuthor` instead of `withMentions`.